test/integration/install/install_test.go

```diff
@@ -429,16 +429,6 @@ func TestControlPlaneResourcesPostInstall(t *testing.T) {
 	testutil.TestResourcesPostInstall(TestHelper.GetLinkerdNamespace(), expectedServices, expectedDeployments, TestHelper, t)
 }
 
-func TestCheckHelmStableBeforeUpgrade(t *testing.T) {
```

Comment:
To explain why this is removed:

This was running essentially `linkerd check` with the new version of the CLI before upgrading the control plane (in particular the CRDs) to that version.  The check was failing because the new version of CRD was not found in the cluster.  I believe that this failure is correct and desirable: before performing the upgrade the cluster is not in the desired state according to the new version.  Therefore I don't think it makes sense to run a check here before performing the upgrade.